src/Engine/ProtoCore/FFI/CLRDLLModule.cs

```diff
@@ -1222,15 +1396,22 @@ public IEnumerable<Attribute> Attributes
             get { return attributes; }
         }
 
-        public FFIClassAttributes(Type type)
+        public FFIClassAttributes(Type type) : this(type, false)
+        {
+        }
+
+        public FFIClassAttributes(Type type, bool isReflectionContext)
```

Comment:
since you're in here anyway ;) can you add summaries to these constructors?
Also - is there any reason to have the new overload public?